lib/Dancer2.pm

```diff
@@ -133,7 +133,7 @@ Dancer2 is easy and fun:
 
     use Dancer2;
     get '/' => sub { "Hello World" };
-    dance;
+    dance; Show support resources after generating new app Show support resources after generating new app
```

Comment:
erm?

Reply:
Interesting, not sure what happened there. Thanks for pointing it out!

Reply:
Fixed, thanks.